devtools.js

```diff
@@ -2463,19 +2484,25 @@ function on_battle(json, battle_api_name) {
 	}
 	calc_damage(result, "先制対潜", d.api_opening_taisen,  f_nowhps, e_nowhps, f_nowhps_c, e_nowhps_c);	// 対潜先制爆雷攻撃.　2016-06-30メンテ明けから追加.
 	calc_damage(result, "開幕雷撃", d.api_opening_atack,   f_nowhps, e_nowhps, f_nowhps_c, e_nowhps_c);	// 開幕雷撃.
-	calc_damage(result, "夜戦砲撃", d.api_hougeki,         f_nowhps, e_nowhps, f_nowhps_c, e_nowhps_c);	// midnight
```

Comment:
why this line was removed?

Reply:
It has been relocated before the new night battle api_n_hougehi1/2 lines (at line 2466) to make all night battle related codes go before all day battle codes

Reply:
i see.